{
    "id": "CVE-2017-10708",
    "product": "n/a",
    "version": "n/a ",
    "vulnerability": [
        "n/a"
    ],
    "description": "An issue was discovered in Apport through 2.20.x. In apport/report.py, Apport sets the ExecutablePath field and it then uses the path to run package specific hooks without protecting against path traversal. This allows remote attackers to execute arbitrary code via a crafted .crash file.",
    "poc": {
        "reference": [
            "https://launchpad.net/bugs/1700573"
        ],
        "github": [
            "No PoCs found on GitHub currently."
        ]
    }
}